### DISCUSSION TITLE
No system settings

### DISCUSSION THREAD

**Discussion by Timtaran** [lxqt/lxqt#2649] [Q&A]
I have installed lxqt on my ubuntu server and there is only `Printers` in `System Setting` category in lxqt-config and startup menu (?)

**Comment by stefonarch** [comment]
Probably there isn't anything else?
I've changed "users&groups" to appear there (where it belongs IMO):

![immagine](https://github.com/user-attachments/assets/cd6dc812-a3d8-4593-b589-bf6ca80f58f5)

**Reply by Timtaran** [reply]
yes, there is no anything else looks like desktop settings

**Comment by tsujan** [comment]
> there is only Printers in System Setting category

That's not a problem. The settings that are related to LXQt are under LXQt Settings, and the only other system setting you have is that of printer, like in my case:

![cc](https://github.com/user-attachments/assets/658b9c56-0d6c-4ae1-8ea6-6f42d5a1b812)

**Reply by Timtaran** [reply]
there is also no lxqt settings, I will send all I have a bit later

**Reply by tsujan** [reply]
Then something basic is wrong about your installation.
